install/kubernetes/helm/istio/charts/security/templates/enable-mesh-mtls.yaml

```diff
@@ -1,4 +1,4 @@
-{{ define "security-default.yaml.tpl" }}
+{{- if .Values.global.mtls.enabled }}
```

Comment:
What is wrong with original logic?

Reply:
We can directly create the resource, check origin logic for this: https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/charts/security/templates/configmap.yaml#L14-#L16